_data/compatibility/mycelium.yaml

```diff
@@ -39,17 +39,17 @@ rbf:
         Receiving Transaction Signaling RBF - When receiving RBF, or any transaction, no unconfirmed transactions show.
 segwit:
   tested:
-    date: "2019-07-24"
+    date: "2021-01-27"
     platforms:
       - iOS
     version: "14"
   features:
     receive:
       p2sh_wrapped: "false"
-      bech32: "false"
+      bech32: "true"  # https://github.com/mycelium-com/wallet-android/issues/425#issuecomment-440792004
       default: "p2pkh"
     send:
-      bech32: "false"
+      bech32: "true" # https://github.com/mycelium-com/wallet-android/issues/425#issuecomment-440792004
```

Comment:
Not possible due to validation error in the UX for me

Reply:
Should this then maybe also be reverted?

Reply:
I downloaded and tested Mycelium, added a commit to update the information on Mycelium's segwit compatibility.